_layouts/default.html

```diff
@@ -69,15 +87,15 @@
                                     </a>
                                 </div>
                                 <!-- {% if site.search.enabled == true %}
-                                <form class="search-container mx-2 my-4 px-3 py-2 d-flex"
```

Comment:
We will need to have the search container still on the developer portal, no?

Reply:
I guess we have not discussed the placement of the search bar/container with the implementation of the new sidebar.
The first level of the sidebar is a bit narrow to hold a search bar/container in my opinion. Perhaps place it in the top right corner in desktop view and in the topbar in mobile view?

Reply:
Yep, that sounds reasonable.

Reply:
Will you do a quick rebase of the branch against `HEAD` of `main` as well?